libraries/matcher.rb

```diff
@@ -7,6 +7,13 @@
                    )
   end
 
+  [:set, :add, :remove].each do |action|
```

Comment:
Probably time to kill off the matchers file. Modern ChefSpec in ChefDK generates these on the fly.